Wiki Updation #3 (#153)
NilPointer-Software/bdfd-wiki: src/premium/awaitedReactions.md

# Awaited reactions
*(for premium bots)*

> **Awaited reactions** are same like [awaited commands](../guides/awaitedCommands.md). Unlike awaited commands which waits for a message, it waits for a reaction instead. 

Reviewer: Should be `Awaited reactions are similar to`, its `wait` not `waits`, and `, they wait for a reaction instead.`

```suggestion
**Awaited reactions** are similar to [awaited commands](../guides/awaitedCommands.md). Unlike awaited commands which wait for a message, they wait for a reaction instead. 
```

![end result](https://i.imgur.com/diskuyv.png)

> > 📝 Reaction roles are not possible at the moment since awaited reactions can only be triggered by author and it gets expired whenever bot goes offline.

Reviewer: Should be `by the author and they expire whenever the bot goes offline`

```suggestion
> 📝 Reaction roles are not possible at the moment since awaited reactions can only be triggered by the author and they expire whenever the bot goes offline.
```

## Getting Started
To create an awaited reaction command, following functions and callback are used :

### $awaitReactions[]
This function is used to await an reaction command.

#### Usage
```
$awaitReactions[<Command name;Reaction>;...]
```
- `command name` - It's the name which will be used inside `$reaction[]` callback.

> - `reaction` - It awaits the given emoji. Emoji must be either in unicode or discord emoji id format.

Reviewer: Should be `or in`

```suggestion
- `reaction` - It awaits the given emoji. Emoji must be either in Unicode or in discord emoji id format.
```

> > 📝 You can group reactions by specifying more *"command name(s)"* and *"reaction(s)"* in `$awaitReactions[]`.\

Reviewer: Drop the `()` around those `s`

```suggestion
> 📝 You can group reactions by specifying more *"command names"* and *"reactions"* in `$awaitReactions[]`.\
```

\

> > ⚠️ In group reactions, when one reaction is used. Then, the other(s) stops working i.e let's say, a command awaits two reactions (✔️ & ❌). If user reacts ✔️ then ❌ stops working.

Reviewer: Should be `when one reaction is used, the others stop working`

```suggestion
> ⚠️ In group reactions, when one reaction is used, the others stop working i.e let's say, a command awaits two reactions (✔️ & ❌). If the user reacts ✔️ then ❌ stops working.
```
